tests/func/test_sync.py

```diff
@@ -2,6 +2,43 @@
 
 from __future__ import unicode_literals
 
+import os
+from textwrap import dedent
+
+
+def test_non_superuser(dev, psql):
+    from sh import ldap2pg
+    c = 'tests/func/ldap2pg.nonsuper.yml'
+    db = 'nonsuper'
+    env = dict(
+        os.environ,
+        PGUSER='nonsuper',
+        PGDATABASE=db,
+    )
+    env.pop('PGDSN', None)
+    myldap2pg = ldap2pg.bake(c=c, _env=env)
+
+    psql(d=db, c=dedent("""\
```

Comment:
@schmandr even if it can't grant CREATE, i check the case where a role effectively own a table.